wandb/sdk/lib/credentials.py

```diff
@@ -0,0 +1,85 @@
+import json
+import os
+from datetime import datetime, timedelta
+from typing import Optional
+
+import requests.utils
+
+from wandb.errors import AuthenticationError
+
+DEFAULT_WANDB_CREDENTIALS_FILE = os.path.expanduser("~/.config/wandb/credentials.json")
```

Comment:
i would use pathlib instead of os for windows.

Reply:
is it possible to provide a different credentials file? some users won't have access to this location.